src/blocks/order-details/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/editor';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+import './style.scss';
+
+/**
+ * Register and run the "Order Details" block.
+ */
+registerBlockType( 'woocommerce-gutenberg-emails/order-details', {
+	title: __( 'Order Details', 'woocommerce-gutenberg-emails' ),
+	category: 'woocommerce-gutenberg-emails',
+	keywords: [ __( 'WooCommerce', 'woocommerce-gutenberg-emails' ) ],
+	description: __(
+		'Display order details in a table.',
+		'woocommerce-gutenberg-emails'
+	),
+
+	/**
+	 * Renders and manages the block.
+	 *
+	 * @param {Object} props
+	 * @return {Object} Editor component.
+	 */
+	edit( props ) {
+		return <Block { ...props } />;
+	},
+
+	/**
+	 * Block content is rendered in PHP, not via save function.
+	 *
+	 * @return {Object} Visible component.
+	 */
+	save() {
+		return <InnerBlocks.Content />;
```

Comment:
`InnerBlocks.Content` doesn't mean anything without using `InnerBlocks` in the edit mode (you got this from the Featured Product block, I'm guessing). That one's more complicated, I would recommend [my recipe block](https://github.com/ryelle/rmb-recipe-block/blob/master/assets/js/recipe-block.js) as a better "intro" to blocks.

Blocks store data in two places, attributes and InnerBlocks content; and saving a block converts that to static HTML - unless it's a dynamic block, which is rendered in PHP (then the save function doesn't do much, and is usually `return null`). `InnerBlocks` lets you add in arbitrary blocks and `InnerBlocks.Content` saves the rendered HTML (it basically lets you pass off handling data to other blocks, we'll want to do this with the `EmailWrapper` block).

So in the Featured Product block, we use an InnerBlocks area for the button, and then save that using `InnerBlocks.Content` so that the PHP rendering can pull that HTML [from `$content`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/php/class-wgpb-block-featured-product.php#L41-L48).

All that's to say we probably don't need to do dynamic rendering for these blocks, I would actually suggest saving to HTML with the placeholders- this way they can be replaced when the email is sent using the code in #22.

Reply:
> you got this from the Featured Product block, I'm guessing

Yes, copy&pasted from there! :smile: 

Thanks for the explanation, I updated it in b50f5307f07fd94cb0e55e0162c278e74e51b865 so it doesn't use `InnerBlocks`, hopefully it makes more sense now. I'm leaving it with a hard-coded _Lorem Ipsum_ message so all the logic implementation is done in a follow-up when fixing #15.